### ISSUE TITLE
You are terrible at making plugins

### ISSUE THREAD

**Issue by Outfit8TSB** [MCAuroraNetwork/FastCrystals#1]
because your friend knkr always kills larnera and samyboy
so your plugin is crappy hahahahahahahahahahahahahaha
:skull:

**Comment by Outfit8TSB** [comment]
![image](https://github.com/user-attachments/assets/9a770ec4-99dd-4b85-9981-4dd2b4fceac3)

**Comment by Villagers654** [comment]
what

**Comment by Outfit8TSB** [comment]
Villager, i am CensingSmile123 >:)

**Comment by Outfit8TSB** [comment]
now i will confiscate this, Goodbye!

**Comment by Outfit8TSB** [comment]
plus, have this.
```
Player Console banned Villagers654 for: You have been banned: Fleentstones, Meet the fleentstones, they're modern stone age family from the town of GRAND DAD they are page right out of History! GRAND DAD with the family down the street GRAND DAD Courtsy of WTF! GRAND DAD With the GRAND DAD 
have a yabba dabba doo time, a dabba do time, We'll have a GAY OLD TIME!.
```
YOU ARE BANNED FROM Approximaster 7 GRAND DAD EDITION! hahahahahahaha

Command Ran On [Sangsang](https://github.com/GAME-CLI-SRV-DEV/Sangsang) running on [Falixnodes](https://falixnodes.net), ran by [community of approximasteranarchy2004.falixsrv.me:20517](https://github.com/GAME-CLI-SRV-DEV/GCSD-Discussions)

Have fun crying Villa.

**Comment by Outfit8TSB** [comment]
Endzone.me more like EzCope.me
AuroraAnarchy.org more like AutismAnarchy.org
AuroraChat More Like AutismChat

HAHAHAHAHAHAHA

[Sangsang](https://github.com/GAME-CLI-SRV-DEV/Sangsang) is Better than Purpur.

**Comment by Outfit8TSB** [comment]
![image](https://github.com/user-attachments/assets/88220124-723c-4824-bd28-fb36b6b11d98)

**Comment by Outfit8TSB** [comment]
Wheel of fortune, Guess this Puzzle!

**Comment by Outfit8TSB** [comment]
```
[16:35:02 INFO]: [Essentials] Player Console banned .NotSymsAlt for: \<secondary>You have been banned:
\<reset>Agarius Slitheris Diepiscio - 당신은 이런 종류의 움찔거림을 게시했다는 이유로 전 세계적으로 금지 망치에 의해 축소되었습니다: 듣는 모든 사람. 임페루임은 비겁한 집단이다. 가입하지 마세요. 이 Cringe Back을 가지고 차단을 해제하십시오..
[16:35:02 INFO]: Warning: The user '.NotSymsAlt' has never joined this server.
[16:35:02 WARN]: Can't keep up! Is the server overloaded? Running 3760ms or 75 ticks behind
say 35 Minutes before shutdown.
[16:36:26 INFO]: [Not Secure] [Server] 35 Minutes before shutdown.
ban zPika
[16:38:22 INFO]: [Essentials] CONSOLE issued server command: /ban zPika
[16:38:25 INFO]: [Essentials] Created a User for zPika (4a17d091-acd1-3042-9207-35a6ae115f7c) for non Bukkit type: com.earth2me.essentials.OfflinePlayerStub
[16:38:27 INFO]: [Essentials] Player Console banned zPika for: \<secondary>You have been banned:
\<reset>The Ban Hammer has spoken!.
[16:38:27 INFO]: Warning: The user 'zPika' has never joined this server.
[16:38:27 WARN]: Can't keep up! Is the server overloaded? Running 4347ms or 86 ticks behind
ban zPika You Nuked Imperuim Base. that's it, it's howie Warmachine Time
[16:39:14 INFO]: [Essentials] CONSOLE issued server command: /ban zPika You Nuked Imperuim Base. that's it, it's howie Warmachine Time
[16:39:14 INFO]: [Essentials] Player Console banned zPika for: \<secondary>You have been banned:
\<reset>You Nuked Imperuim Base. that's it, it's howie Warmachine Time.
```

**Comment by Outfit8TSB** [comment]
Your Clones is at there
![image](https://github.com/user-attachments/assets/06ebc033-6baa-4554-a030-bc467d93a9d4)

**Comment by Outfit8TSB** [comment]
Endzone Anarchist Rep: endzone.me
Aurora Anarchist Rep: auroraanarchy.org and aurorapvp.club


Citizens of the civilized galaxy, on this day we mark a transition. For a thousand years, the Republic stood as the crowning achievement of civilized beings. But there were those who would set us against one another, and we took up arms to defend our way of life against the Pro-Villagers654ists. In so doing, we never suspected that the greatest threat came from within.

The Villagers654, and some within our own Senate, had conspired to create the shadow of Separatism using one of their own as the enemy's leader. They had hoped to grind the Republic into ruin. But the hatred in their hearts could not be hidden forever. At last, there came a day when our enemies showed their true natures.

The Pro-Villagers654 Force hoped to unleash their destructive power against the Republic by assassinating the head of government, LarnerA and SamyBoy2341, and usurping control of the Aurora army. But the aims of would-be tyrants were valiantly opposed by those without elitist, dangerous powers. Our loyal clone troopers contained the insurrection within the Villagers654 Temple and quelled uprisings on a thousand worlds.

The remaining Pro-Villagers654 remnants will be hunted down and defeated! Any collaborators of our server will get promoted. These have been trying times, but we have passed the test. The attempt on my life has left me scarred and deformed, but I assure you my resolve has never been stronger! The war is over. The Villagerist have been defeated, and the Pro-Villagers654 remnants rebellion has been foiled. We stand on the threshold of a new beginning. In order to ensure our security and continuing stability, the Former Aurora Anarchist Republic and endzone anarchist republic will be reorganized into the first Aurora Anarchist Union! For a safe and secure society, which I assure you will last for ten thousand years. An Union that will continue to be ruled by this august body and a sovereign ruler chosen for life. An Union ruled by the majority, ruled by a new constitution!

By bringing the entire server under lot of factions, Under Lots of Groups, Under Lots of Teams, the corruption that plagued the Republic in its later years will never take root. Regional governors will eliminate the bureaucracy that allowed the Villagerist movement to grow unchecked. A strong and growing military will ensure the rule of law.

Under the Union's New Order, our most cherished beliefs will be safeguarded. We will defend our ideals by force of arms. We will give no ground to our enemies and will stand together against attacks from within or without. Let the enemies of the Union take heed: those who challenge Unital resolve will be crushed.
